Bluepill-runner/BluepillRunnerTests/BPRunnerTests.m

```diff
@@ -30,7 +30,6 @@ - (void)setUp {
     NSString *hostApplicationPath = [BPTestHelper sampleAppPath];
     NSString *testBundlePath = [BPTestHelper sampleAppNegativeTestsBundlePath];
     self.config = [BPConfiguration new];
-    self.config.program = BP_MASTER;
```

Comment:
No longer necessary as we are not making a `mutableCopy` of the `BPConfiguration` instance.